libpod/sqlite_state_internal.go

```diff
@@ -0,0 +1,288 @@
+package libpod
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/podman/v4/libpod/define"
+	"github.com/sirupsen/logrus"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// Initialize all required tables for the SQLite state
+func sqliteInitTables(conn *sql.DB) (defErr error) {
+	// Technically we could split the "CREATE TABLE IF NOT EXISTS" and ");"
+	// bits off each command and add them in the for loop where we actually
+	// run the SQL, but that seems unnecessary.
+	const dbConfig = `
+        CREATE TABLE IF NOT EXISTS DBConfig(
+                Id            INTEGER PRIMARY KEY NOT NULL,
+                SchemaVersion INTEGER NOT NULL,
+                Os            TEXT    NOT NULL,
```

Comment:
Can we call it `ID` and `OS`?  Maybe just my personal preference but `Id` and `Os` always look wrong to me.